PIconnect/time.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+from typing import Union
+
+from PIconnect.AFSDK import AF
+
+
+def to_af_time_range(start_time: Union[str, datetime], end_time: Union[str, datetime]):
```

Comment:
Could you move these type hints to comments for now? I would like to merge this for version 0.9, but that version would still need to support Python 2.7. After that release I want to add type hints as much as possible.

Reply:
ok